src/main/scala/edu/ie3/simona/main/RunSimonaStandalone.scala

```diff
@@ -56,9 +61,38 @@ object RunSimonaStandalone extends RunSimona[SimonaStandaloneSetup] {
       case SimonaEnded(successful) =>
         simonaSim.terminate()
 
+        val config = SimonaConfig(simonaSetup.typeSafeConfig).simona.output
+
+        config.sink.csv.map(_.zipFiles).foreach { zipFiles =>
+          if (zipFiles) {
+            val rawOutputPath =
+              Path.of(simonaSetup.resultFileHierarchy.rawOutputDataDir)
+
+            rawOutputPath.toFile.listFiles().foreach { file =>
+              val fileName = file.getName
+              val archiveName = fileName.replace(".csv", "")
+              val filePath = rawOutputPath.resolve(fileName)
+
+              val compressFuture =
+                FileIOUtils
+                  .compressFile(filePath, rawOutputPath.resolve(archiveName))
+                  .asScala
+              compressFuture.onComplete {
+                case Success(_) =>
+                  FileIOUtils.deleteRecursively(filePath)
+                case Failure(exception) =>
+                  logger.error(
+                    s"Compression of output file to '$archiveName' has failed. Keep raw data.",
+                    exception,
+                  )
+              }
+              Await.ready(compressFuture, 5.minutes)
```

Comment:
Do we / you have any feeling about a proper value here. We had some huge simulations in the past where a single result file had up to 20 Gb. Even this is seldom and one can adapt this value in these cases, we should think about it here.

Reply:
We could either increase this time (for example to 1 hour or more) or we could make this cofigurable via the provided config.

Reply:
Thats the point. I rather would not go via config. But I'm not sure if 5 minutes are sufficient. Maybe 15 will be a good start? What do you think?

Reply:
Do you know how long compressing can take if we have a 20 GB file?

Reply:
That was my initial question :) 

I have no idea. Depends of course on the machine, etc... maybe you can find out what would be an good estimation? Otherwise I would go with 15 minutes.